File: LVJ_AdminVersion/ModelosPersistencia/Usuario.cs
class Usuario
{
    public int id { get; set; }
    public string nome { get; set; }
    public string email { get; set; }
    public string senha { get; set; }
    public string endereco { get; set; }
    public string cep { get; set; }
    public string cpf { get; set; }
    public bool admin { get; set; }

    public Usuario(int id, string nome, string email, string senha, string endereco, string cep, string cpf, bool admin)
    {
        this.id = id;
        this.nome = nome;
        this.email = email;
        this.senha = senha;
        this.endereco = endereco;
        this.cep = cep;
        this.cpf = cpf;
        this.admin = admin;
    }

    public override string ToString()
    {
        if(admin == true){
            return $"[{id}] - {nome} \n\tEmail: {email} \n\tSenha: {senha} \n";
        } else {
            return $"[{id}] - {nome} \n\tEmail: {email} \n\tSenha: {senha} \n\tEndereço: {endereco} \n\tCEP: {Convert.ToDouble(cep).ToString(@"00\.000-000")} \n\tCPF: {Convert.ToDouble(cpf).ToString(@"000\.000\.000-00")} \n";
        }
    }
}

File: LVJ_AdminVersion/ModeloPersistênciaPython/Usuario.py
class Usuario:
    @property
    def id (self):
        return self._id
    
    @id.setter
    def id(self, novo_id):
        self._id = novo_id

    
    @property
    def nome(self):
        return self._nome
    
    @nome.setter
    def nome(self, novo_nome):
        self._nome = novo_nome

    
    @property
    def email (self):
        return self._email
    
    @email.setter
    def email(self, novo_email):
        self._email = novo_email

    
    @property
    def endereco(self):
        return self._endereco
    
    @endereco.setter
    def endereco(self, nova_endereco):
        self._endereco
            
    
    @property
    def cep(self):
        return self._cep
    
    @cep.setter
    def cep(self, nova_cep):
        self._cep
            
    
    @property
    def cpf(self):
        return self._cpf
    
    @cpf.setter
    def cpf(self, nova_cpf):
        self._cpf
            
    
    @property
    def admin(self):
        return self._admin
    
    @admin.setter
    def admin(self, nova_admin):
        self._admin
            
    def __str__(self):
        if (admin == true):
            return f"{id} - {nome} \n\tEmail: {email} \n\tSenha: {senha} \n"
        else:
              return f"{id} - {nome} \n\tEmail: {email} \n\tSenha: {senha} \n\tEndereço: {endereco} \n\tCEP: {CEP} \n\tCPF: {CPF} \n"
